### DISCUSSION TITLE
原partner-gitlab.mioffice.cn上其他项目会迁移到GitHub吗？

### DISCUSSION THREAD

**Discussion by zbwu** [MiRoboticsLab/cyberdog_ros2#15] [General]
原partner-gitlab.mioffice.cn上其他项目会迁移到GitHub吗？

还是只是ROS2相关的放到GitHub，其他继续使用原来的代码库？

**Comment by homalozoa** [comment]
会逐步迁移过来的。ROS 2因为主要是我维护的，所以迁移效率高一些。
